spec/runtime/executable_spec.rb

```diff
@@ -8,142 +8,9 @@
     G
   end
 
```

Comment:
It seems like this file was primarily testing `bundle install --binstubs`, so most of its specs have been removed.